### ISSUE TITLE
DOM isn't finished rendering when initial `filteringEnd` is triggered

### ISSUE THREAD

**Issue by gkrinc** [giotiskl/filterizr#103]
I'm trying to get a list of visible items using `$('.filtr-item:not(.filteredOut)')` but the `filteredOut` class hasn't been written to the DOM when the initial `filteringEnd` event is triggered. The items are hidden though, `opacity: 0`, etc.

I have to set an arbitrary timeout to ensure the class is present like this:

```
$filterizd.on('filteringEnd', function () {
    setTimeout(function () {
        // At this point the `filteredOut` class `should` be applied to the grid items that are hidden
    }, 250);
});
```

Is there a more precise way to get a list of the filtered out element from filterizr?

**Comment by giotiskl** [comment]
All timing issues with events should be resolved in v2.2.0 onwards. Also many new features have been added.

**Comment by Xoxsossardoii** [comment]
Hello there and thank you for the wonderful library! It works wonders except for the filteringEnd callback. I have the same problem as above but only on Blink-based browsers (Firefox and all of its clones works without a problem). 

What I'm trying to achieve is to remove the filtered out items from the tabindex. My code works on all browsers but when I press on the "All" button, Blink-based browsers doesn't update the tabindex until I press a second time. I can fix the problem with a setTimeout of 500ms but it's not ideal.

In short, filteringEnd works good on all filters except "All", where there is short delay (around 400-500ms). Only on Blink-based browsers.

Any idea what might be the problem? I'm using v2.2.3

Here's my code:
```
let filtrItem = document.getElementsByClassName("filtr-item");
let fL = filtrItem.length;

for (let i = 0; i <= fL; i++) {
   if (filtrItem[i].classList.contains("filteredOut")) {
      filtrItem[i].setAttribute("tabindex", "-1");
   } else {
      filtrItem[i].setAttribute("tabindex", "0");
   }
}
```
